index.md

```diff
@@ -1,5 +1,63 @@
-# Welcome to VideOut!
+# Welcome to Videout!
 
-VideOut is a language design to edit videos/gifs/photos through coding. This way programmers that have
+Videout is a language design to edit *videos/gifs* through coding. This way programmers that have
 no experience with media editing programs, can do projects that involve said task. The language was 
-designed as an asignment for the Programming Langages course at UPRM. 
+designed as an assignment for the Programming Language course at UPRM. 
+
+The video editing tools that this language allows you to use are: 
```

Comment:
```
## Installation
```
Required software and a short explanation.
Python dependencies included. However, these may or may not need explanations

Reply:
Done, tho right now the info it has is the same as the readme .md. It will be edited tho, I just added it as a template of what should that part be about.